src/main/java/org/codehaus/mojo/license/AbstractLicenseNameMojo.java

```diff
@@ -180,6 +180,14 @@ public abstract class AbstractLicenseNameMojo extends AbstractLicenseMojo {
     @Parameter
     protected Map<String, String> extraTemplateParameters;
 
+    /**
+     * Ignore the last date for copyright year range, preferred by some legal perspectives.
```

Comment:
I'm not sure if reason should be given here - end of sentence ` ... preferred by some legal perspectives`

We can simply say what parameter does.